components/frontend/src/dashboard/ExportCard.js

```diff
@@ -0,0 +1,67 @@
+import { Card, List } from 'semantic-ui-react';
+import { childrenPropType, datePropType, reportPropType } from '../sharedPropTypes';
+import { bool, string } from 'prop-types';
+import './ExportCard.css';
+
+function ExportCardItem({ children, url }) {
+    const item = children;
+    return url ? <List.Item as="a" href={url}>{item}</List.Item> : <List.Item>{item}</List.Item>;
+}
+ExportCardItem.propTypes = {
+    children: childrenPropType,
+    url: string,
+}
+
+export function ExportCard({ report, last_update, report_date, is_overview = false }) {
```

Comment:
The code base still has a mixture of camelCase and snake_case names for Javascript variables, parameters, and props, but we prefer camelCase. So please rename the props to lastUpdate, reportDate and isOverview.